python/ray/serve/_private/deployment_state.py

```diff
@@ -1191,6 +1192,8 @@ def __repr__(self):
 class DeploymentState:
     """Manages the target state and replicas for a single deployment."""
 
+    FORCE_STOP_UNHEALTHY_REPLICAS = RAY_SERVE_FORCE_STOP_UNHEALTHY_REPLICAS
```

Comment:
is this for style to have instance attribute?

Reply:
it's for testing so I can just set the static field to test both behaviors

Reply:
@edoakes why can't you patch top-level variable like this `patch(ray.serve._private.RAY_SERVE_FORCE_STOP_UNHEALTHY_REPLICAS, 0)`?